### PR TITLE
Added/Updating Logic Apps Management Client endpoints

### PR DESCRIPTION
**Pull request by refortie**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by daviburg** [comment]
Reid, are you able to add Logic App engineers as reviewers for this PR? We should get all the authors of the operations you are adding coverage for to review their piece.

**Comment by refortie** [comment]
@daviburg 
> Reid, are you able to add Logic App engineers as reviewers for this PR? We should get all the authors of the operations you are adding coverage for to review their piece.

I'm not super familiar with github's ui, I'll see if I can figure out how to add them

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json [hunk 1]
```diff
@@ -0,0 +1,1918 @@
...
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "parameters": {
+        "locationParameter": {
+            "name": "location",
+            "description": "The location.",
```
Comment (daviburg) [inline specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: consistent ending of lines with or without dot

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json [hunk 2]
```diff
@@ -0,0 +1,1918 @@
...
+                            "$ref": "#/definitions/ApiConnectionDefinition"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "tags": [
+                    "Connections"
+                ],
+                "summary": "Update an existing connection",
```
Comment (daviburg) [inline specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: should all operations have a summary like this one? delete below doesn't

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json [hunk 3]
```diff
@@ -0,0 +1,1918 @@
...
+                "operationId": "Connections_Update",
+                "description": "Update a connection",
```
Comment (daviburg) [inline specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: Verb is with present tense below, i.e. Deletes
Should this here be Updates then?

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json [hunk 4]
```diff
@@ -0,0 +1,1918 @@
...
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/customApis/{apiName}/move": {
+            "post": {
+                "tags": [
+                    "Custom APIs"
+                ],
+                "summary": "Moves the Custom API",
```
Comment (daviburg) [inline specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: consistent capitalization for custom in custom API

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json [hunk 5]
```diff
@@ -0,0 +1,1918 @@
...
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/connectionGateways/{connectionGatewayName}": {
+            "get": {
+                "tags": [
+                    "Connection Gateways"
+                ],
+                "summary": "Gets a specific gateway",
+                "operationId": "ConnectionGateways_Get",
+                "description": "Gets a specific gateway for under a subscription and in a specific resource group",
```
Comment (daviburg) [inline specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: "for under" is that proper English? above was just "under"

Comment (refortie) [reply specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: Nope, that is not proper English. Fixed

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json (hunk 2 above)
Comment (refortie) [reply specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: Yes, they all should. Will doublecheck all of them

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json [hunk 6]
```diff
@@ -0,0 +1,1918 @@
...
+                }
+            }
+        },
+        "ConsentLinkCollection": {
+            "description": "Collection of consent links",
+            "type": "object",
+            "properties": {
+                "value": {
+                    "description": "Collection of resources",
+
```
[the remark this answers is not in the document]
Comment (refortie) [reply specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: Fixed

File: specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json [hunk 7]
```diff
@@ -0,0 +1,1918 @@
...
+                    "description": "Resource type",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "location": {
+                    "description": "Resource location",
+                    "type": "string"
+                },
+                "etag": {
+                    "description": "Resource etag",
```
[the remark this answers is not in the document]
Comment (refortie) [reply specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json]: I believe so

**Comment by refortie** [comment]
Need the examples files. Going to close this till those are done

**Pull request closed without merging**